inst/extdata/energy/A24.globaltech_coef.csv

```diff
@@ -0,0 +1,8 @@
+# File: A24.globaltech_coef.csv,,,,,
```

Comment:
Here and in files below, please remove these stupid Excel commas at end of header lines